### PR TITLE
Markets in suspension can be resumed or terminated after a protocol u…

### PR DESCRIPTION
**Pull request by vega-paul**
…pgrade restarts the network

### CODE REVIEW

**Review by gordsport — CHANGES_REQUESTED** [review]
Please also add the new AC to:

- https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/features.json

BEFORE it is merged - if applicable to a cosmic elevator feature

File: protocol/0028-GOVE-governance.md
Comment (Jiajia-Cui) [inline protocol/0028-GOVE-governance.md]: Markets is suspended by governance
 
 can be split into 2 ACs

**Review by Jiajia-Cui — DISMISSED** [review]
LGTM

**Comment by vega-paul** [comment]
@gordsport https://github.com/vegaprotocol/specs/pull/1868

**Review by gordsport — APPROVED** [review]
Approving as the content has not changed since the last approval

**Pull request merged**
